### PR TITLE
Some missing translations enable

### PR DESCRIPTION
**Pull request by kamrulhasanj**
**Description**
Some missing translations are enabled in TT4

### CODE REVIEW

File: patterns/footer-portfolio.php [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
 <!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
 <div class="wp-block-group"><!-- wp:paragraph {"fontSize":"small"} -->
-<p class="has-small-font-size">© 2024 Twenty Twenty Four</p>
+<p class="has-small-font-size"><?php echo esc_html_x( '© 2024 Twenty Twenty Four', '', 'twentytwentyfour' ); ?></p>
```
Comment (huzaifaalmesbah) [inline patterns/footer-portfolio.php:61]: use like that `© 2024 Twenty Twenty-Four` and give context

File: patterns/footer-portfolio.php [hunk 2]
```diff
@@ -58,7 +58,7 @@
...
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph {"fontSize":"small"} -->
```
Comment (huzaifaalmesbah) [inline patterns/footer-portfolio.php:64]: You also follow the footer.php, and enable the translate 
`<p class="has-small-font-size">Designed with <a rel="nofollow" href="https://wordpress.org">WordPress</a></p>`

**Review by MaggieCabrera — CHANGES_REQUESTED** [review]
Let's hold off with the translations until all the patterns have been reviewed

File: patterns/footer-portfolio.php (hunk 1 above)
Comment (juanfra) [reply patterns/footer-portfolio.php:61]: Twenty Twenty Four is probably something we don't want to translate. This string should be processed so the copyright year comes from a variable and not part of the translatable string.

Comment (carolinan) [reply patterns/footer-portfolio.php:61]: I think this should be the site title block.

File: patterns/footer-portfolio.php (hunk 2 above)
Comment (juanfra) [reply patterns/footer-portfolio.php:64]: Please check these lines: https://github.com/WordPress/twentytwentyfour/blob/trunk/patterns/footer.php#L16-L24

**Comment by MaggieCabrera** [comment]
Closed in favor of https://github.com/WordPress/twentytwentyfour/pull/671

**Pull request closed without merging**